golang/pkg/cli/config_file.go

```diff
@@ -467,8 +473,8 @@ func scanPort(portNum uint) uint {
 	for scanner.Scan() {
 		newPort, err := strconv.ParseUint(scanner.Text(), ParseBase, ParseBitSize)
 		if err != nil || (newPort > 0 && newPort <= 1023) || newPort == 0 {
-			fmt.Fprintf(os.Stderr, "you typed invalid port number:\n")
-			fmt.Fprintf(os.Stdout, "type another port: ")
+			log.Error().Err(err).Msg("")
+			log.Info().Msg("Type another port: ")
```

Comment:
Can I type the port right after the colon or into new line? I don't like typing into a new line in case of an input.